### DISCUSSION TITLE
Parameters out of bounds

### DISCUSSION THREAD

**Discussion by nbungart1999** [phoebe-project/phoebe2#487] [Questions]
So for context I only have 1 lc dataset but ebai ran fine and produced a graph. I am now just adopting my estimators and then trying to plot after all estimators have been adopted. However, after adopting ebai, numerous parameters have gone out of bounds which will not allow me to run_compute before plotting. How can I fix this issue?
![image](https://user-images.githubusercontent.com/83960306/124640301-38b97300-de5b-11eb-81fb-30fac7aa4ed6.png)

**Comment by kecnry** [comment]
The estimators only return proposed values for parameters that they are designed to estimate, while keeping all other values fixed.  This can result in leaving the range of atmosphere tables or even resulting in unphysical systems.  So you may need to vary other parameters to ensure you still have a physical/reasonable system before being able to run a forward model within the atmosphere tables.  

If you need to compute a forward model before fixing the parameters that are causing the out-of-bounds errors (in order to use optimizers or just see if the eclipse positions/widths look better), then you can fallback on using blackbody atmospheres (`atm='blackbody'`) and manual limb-darkening (`ld_mode='manual'` and `ld_mode_bol='manual'`) and it should run without errors... but you'll probably want to revert that later once you find your way towards the true values.

**Reply by nbungart1999** [reply]
Ok, so I've been playing around with some values to try and make a reasonable system but I am still getting the same general issue of parameters out of bounds. However, the main thing I noticed is the out-of-bounds teff, it isn't the primary or secondary teff but rather just a teff, what does this mean? Shouldn't the system be specifying primary or secondary?

**Reply by nbungart1999** [reply]
Wait nvm, after printing the bundle parameters I can identify which teff it is
